src/e2e/java/teammates/e2e/cases/InstructorNotificationsPageE2ETest.java

```diff
@@ -0,0 +1,65 @@
+package teammates.e2e.cases;
+
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.AccountAttributes;
+import teammates.common.datatransfer.attributes.NotificationAttributes;
+import teammates.common.util.AppUrl;
+import teammates.common.util.Const;
+import teammates.e2e.pageobjects.InstructorNotificationsPage;
+
+/**
+ * SUT: {@link Const.WebPageURIs#INSTRUCTOR_NOTIFICATIONS_PAGE}.
+ */
+public class InstructorNotificationsPageE2ETest extends BaseE2ETestCase {
+    private AccountAttributes account;
+    private NotificationAttributes notificationToMarkAsRead;
+
+    @Override
+    protected void prepareTestData() {
+        testData = loadDataBundle("/InstructorNotificationsPageE2ETest.json");
+        removeAndRestoreDataBundle(testData);
+
+        account = testData.accounts.get("INotifications.instr");
+        notificationToMarkAsRead = testData.notifications.get("notification3");
+    }
+
+    @Test
+    @Override
+    public void testAll() {
+        AppUrl notificationsPageUrl = createFrontendUrl(Const.WebPageURIs.INSTRUCTOR_NOTIFICATIONS_PAGE);
+        InstructorNotificationsPage notificationsPage = loginToPage(notificationsPageUrl, InstructorNotificationsPage.class,
+                account.getGoogleId());
+
+        ______TS("verify that only the correct notifications are shown");
```

Comment:
Would it be clearer if we add a bit more description here? For example:
```suggestion
        ______TS("verify that only active notifications with correct target user are shown");
```